model_analyzer/record/types/inter_token_latency_p25.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
```

Comment:
is this needed?

Reply:
I don't know. They are the first line in all of the `.py` files